apps/anoma_node/lib/examples/e_shard.ex

```diff
@@ -0,0 +1,868 @@
+defmodule Anoma.Node.Examples.EShard do
+  @moduledoc """
+  I contain examples on how to interact with the Shard module.
+  """
+
```

Comment:
In general we can remove the word `test_` from all names, we quality import things, test is a redundant word here